### DISCUSSION TITLE
Connectors for 24V/5V power

### DISCUSSION THREAD

**Discussion by mDubovsky** [pilotak/eth-io#6] [Ideas]
Before inserting the PCB into the case and soldering the terminals you most likely want to test the PCB so that you don't have to break the case if anything is wrong.
This is not easily done as there are no other connectors for 5V/24V power. If we add connectors such as the one used for the LED in V2 the initial testing would be a lot easier. 
If you opt to not test the PCB before inserting into the case you can just omit these so it's just an optional choice for people that want to do the test.

Illustrative photo not indicative of the real placement. Just to illustrate the kind of connector I mean:
![conn](https://user-images.githubusercontent.com/9304179/208426167-b968300e-00e2-4be3-b9f4-ac68d2444795.jpg)

**Comment by pilotak** [comment]
it can be powered via USB of the ESP32 but of course the header can be added too

**Reply by mDubovsky** [reply]
@pilotak  Just to further clarify, with all the ideas I am also volunteering to help. So if you think something I mention is a good idea but you don't think you'll have the time to implement I can give it a shot and then just run the pull request by you for approval/further suggestions.

**Reply by pilotak** [reply]
done in 28b9ccb64864791d090ab6205445d9660984af67
